quickstarts/Atlassian JIRA

```diff
@@ -0,0 +1,46 @@
+id: c5c5dd30-dcdf-46b6-9412-f9a1bba5a600
```

Comment:
As discussed this will need to be within it's own folder so you should create a new folder under `quickstarts`:

`quickstarts/atlassian-jira`

Then this file should be renamed to `config.yml` in that folder

so it will be `quickstarts/atlassian-jira/config.yml`